src/Schema/Grammar.php

```diff
@@ -93,6 +93,16 @@ public function compileIndexListing()
         return 'select index_name as `index_name` from information_schema.indexes where table_schema = \'\' and table_name = ?';
     }
 
+    /**
+     * Compile the query to determine the list of foreign keys.
+     *
+     * @return string
+     */
+    public function compileForeignListing()
```

Comment:
Would it be possible to change this to `compileForeignKeys` which will align it with [other database grammars](https://github.com/laravel/framework/blob/eb948833d513088249d73775586d8fe0ccf48158/src/Illuminate/Database/Schema/Grammars/SQLiteGrammar.php#L155)?

Reply:
Yes, doing now, will push in 5